utils/shared.js

```diff
@@ -693,6 +693,51 @@ const swapObjKeysAndValues = (object) => {
 
 const batchLimit = 500;
 
+const getUserProfile = async (req, res, uid) => {
+
+    if(req.method !== 'GET') {
+        return res.status(405).json(getResponseJSON('Only GET requests are accepted!', 405));
+    }
+
+    const { retrieveUserProfile } = require('./firestore');
+    let responseProfile = await retrieveUserProfile(uid);
+
+    if(responseProfile instanceof Error){
+        return res.status(500).json(getResponseJSON(responseProfile.message, 500));
+    }
+
+    console.log(responseProfile);
```

Comment:
A bit context about the log will be helpful, like other updated log messages.

Reply:
This shouldn't be there, it was from troubleshooting... Removing